wire/gen_peer_wire_csv

```diff
@@ -63,6 +63,28 @@ funding_signed,32,signature,64
 funding_locked,36
 funding_locked,0,channel_id,32
 funding_locked,32,next_per_commitment_point,33
+open_channel2,56
```

Comment:
This is OK for now, but I think we're going to have to move to a system where we have experimental CSV files (which get compiled in if EXPERIMENTAL defined).

Reply:
I went back and forth on whether or not to include these. Really they're only here because I needed something to test with :cactus: